src/Dockerfile

```diff
@@ -1,64 +1,74 @@
 FROM docker.io/library/centos:7@sha256:285bc3161133ec01d8ca8680cd746eecbfdbc1faa6313bd863151c4b26d7e5a5
 
 ENV VERSION=%%VERSION%% \
-    HELM_VERSION=%%HELM_VERSION%% \
+    HELM2_VERSION=%%HELM2_VERSION%% \
+    HELM3_VERSION=%%HELM3_VERSION%% \
     KUSTOMIZE_VERSION=%%KUSTOMIZE_VERSION%% \
-    IMAGE_CLEANUP_VERSION=%%IMAGE_CLEANUP_VERSION%% \
+    SEISO_VERSION=%%SEISO_VERSION%% \
     KUBEVAL_VERSION=%%KUBEVAL_VERSION%% \
     ARCHIVE=%%ARCHIVE%% \
     SHA256SUM=%%SHA256SUM%% \
-    HELM_SHA256SUM=%%HELM_SHA256SUM%% \
+    HELM2_SHA256SUM=%%HELM2_SHA256SUM%% \
+    HELM3_SHA256SUM=%%HELM3_SHA256SUM%% \
     KUSTOMIZE_SHA256SUM=%%KUSTOMIZE_SHA256SUM%% \
-    IMAGE_CLEANUP_SHA256SUM=%%IMAGE_CLEANUP_SHA256SUM%% \
+    SEISO_SHA256SUM=%%SEISO_SHA256SUM%% \
     KUBEVAL_SHA256SUM=%%KUBEVAL_SHA256SUM%% \
     SOPS_VERSION=%%SOPS_VERSION%% \
     SOPS_RELEASES_URL="https://github.com/mozilla/sops/releases/download" \
     OKD_RELEASES_URL="https://github.com/openshift/origin/releases/download" \
-    HELM_RELEASES_URL="https://storage.googleapis.com/kubernetes-helm" \
+    HELM_RELEASES_URL="https://get.helm.sh" \
     KUSTOMIZE_RELEASES_URL="https://github.com/kubernetes-sigs/kustomize/releases/download" \
-    IMAGE_CLEANUP_RELEASES_URL="https://github.com/appuio/image-cleanup/releases/download" \
+    SEISO_RELEASES_URL="https://github.com/appuio/seiso/releases/download" \
     KUBEVAL_RELEASES_URL="https://github.com/instrumenta/kubeval/releases/download" \
     JQ_URL="https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64" \
     OC_PLUGINS_REPO="https://github.com/appuio/oc-plugins" \
     KUBECTL_PLUGINS_PATH="/opt/kube/plugins"
```

Comment:
As for the deprecation of `image-cleanup`, we also need to update the deprecation notice in [appuio/oc-plugins:/cleanup/cleanup.sh](https://github.com/appuio/oc-plugins/blob/master/cleanup/cleanup.sh#L97). I'll do a PR for that, if you don't mind. And close https://github.com/appuio/oc-plugins/issues/7 with a helpful migration note. :bulb:

Reply:
I updated the oc plugin's deprecation notice in https://github.com/appuio/oc-plugins/pull/10. Please review!

Reply:
I closed issue appuio/oc-plugins#7 with appropriate references.